browser/net/brave_network_audit_browsertest.cc

```diff
@@ -220,7 +220,8 @@ class BraveNetworkAuditTest : public InProcessBrowserTest {
 
   void SetUpCommandLine(base::CommandLine* command_line) override {
     base::FilePath source_root_path;
-    base::PathService::Get(base::DIR_SRC_TEST_DATA_ROOT, &source_root_path);
+    ASSERT_TRUE(base::PathService::Get(base::DIR_SRC_TEST_DATA_ROOT,
```

Comment:
can we replace all these with `GetChecked`?

Reply:
As I understand, upstream avoids `CHECK`s in tests whenever `ASSERT` suffices, so test failures can be handled better, so I would prefer to keep this form of test failure communication here. I also didn't find `GetChecked` in `PathServices`.

Reply:
Fixed in d4f8d12cabc721f8c060c247c3be10cbddc2ccbf. I was unaware of this function, and actually it is being used in tests. Thanks for pointing it out.